department-of-veterans-affairs/abd-vro
# Mcp 2115 - Clean up sec-lighthouse-api integrationTest.

IntegrationTest properties clean up

## What was the problem?
See comments in PR #817.
TODO: run the tests with `@ActiveProfiles("integrationTest")` and remove unnecessary settings below
#   See application-test.properties for an example.
#   May even be able to use `@ActiveProfiles({"test","integrationTest"})` to remove more settings

Associated tickets or Slack threads:
MCP-2115

## How does this fix it?[^1]
Removed the unnecessary setting in the property file for integration test, and kept lh related as the credential values used in test are faked and cannot be used to connect Lighthouse FHIR API sandbox used in the integration test.

## How to test this PR
- Step 1. build app and launch it. The app should be up in local dev environment.
- Step 2. run ./gradlew integrationTest. The test should be successful. Jacco coverage report should show a decent coverage.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
